src/sentry/api/helpers/group_index/update.py

```diff
@@ -136,20 +136,25 @@ def get_current_release_version_of_group(group: Group, follows_semver: bool = Fa
     """
     current_release_version = None
     if follows_semver:
-        try:
-            # This sets current_release_version to the latest semver version associated with a group
-            associated_release_id = GroupRelease.objects.filter(
-                project_id=group.project.id, group_id=group.id
-            ).values_list("release_id")
-
-            current_release_version = (
-                get_semver_releases(group.project)
-                .filter(id__in=associated_release_id)
-                .values_list("version", flat=True)[:1]
-                .get()
-            )
-        except Release.DoesNotExist:
-            pass
+        if not features.has(
```

Comment:
@JoshFerge does this look better? I'm uncertain if I understood perfectly your first comment.

Reply:
yeah exactly